regex-syntax/src/hir/literal.rs

```diff
@@ -172,8 +172,9 @@ impl Extractor {
         use crate::hir::HirKind::*;
 
         match *hir.kind() {
-            Empty | Look(_) => Seq::singleton(self::Literal::exact(vec![])),
-            LookAround(_) => Seq::infinite(),
+            Empty | Look(_) | LookAround(_) => {
+                Seq::singleton(self::Literal::exact(vec![]))
+            }
```

Comment:
This changes requires updating some of the unit tests that check explicitly the extracted literals.
(which is the reason the pipeline currently fails)

Reply:
Oops, I only ran the automata tests locally, sorry for that